instrumentation/instabeego/http_client_handler.go

```diff
@@ -0,0 +1,24 @@
+// (c) Copyright IBM Corp. 2023
+
+//go:build go1.18
+// +build go1.18
+
+package instabeego
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/beego/beego/v2/client/httplib"
+	instana "github.com/instana/go-sensor"
+)
+
+// InstrumentRequest wrap the original BeegoHTTPRequest transport with instana.RoundTripper().
+func InstrumentRequest(sensor *instana.Sensor, req *httplib.BeegoHTTPRequest) {
```

Comment:
Please, use `instana.TracerLogger` interface instead of the concrete implementation `*instana.Sensor`.
This makes it easier for us (or any SDK) to be extended and/or modified without causing breaking changes too easily.

Reply:
Thank you for the info. Made the changes